build.assets/tooling/cmd/build-apt-repos/aptly.go

```diff
@@ -0,0 +1,200 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+// This provides wrapper functions for the Aptly command. Aptly is written in Go but it doesn't appear
+// to have a good binary API to use, only a CLI tool and REST API.
```

Comment:
We probably can. If it provides a good interface to implement what we need to, I agree that would be a superior approach to calling a binary. @fheinecke Can you take a look if we can import the library directly and use https://pkg.go.dev/github.com/aptly-dev/aptly@v1.4.0/deb instead of invoking binaries?

Reply:
Ended up not going this route as the docs and project structure make it look like Aptly is _really_ not designed to be used as a library by another application. Doing so would add significant effort required.